code/lib/cli-storybook/src/codemod/helpers/csf-factories-utils.ts

```diff
@@ -75,12 +75,38 @@ export function removeExportDeclarations(
 }
 
 export function getConfigProperties(
-  exportDecls: Record<string, t.VariableDeclarator | t.FunctionDeclaration>
+  exportDecls: Record<string, t.VariableDeclarator | t.FunctionDeclaration>,
+  options: { configType: 'main' | 'preview' }
 ) {
   const properties = [];
 
   // Collect properties from named exports
   for (const [name, decl] of Object.entries(exportDecls)) {
+    // only include real preview exports to definePreview factory
+    if (
+      options.configType === 'preview' &&
+      ![
+        'decorators',
```

Comment:
Can you maybe make this into a type and have that type be connected with the actual annotations?
```
const annotationNames: (keyof NormalizedProjectAnnotations)[] = [
  'decorators',
  // ...
];
```